orchestrator/api/codegraph.py

```
@@ -332,23 +328,68 @@ async def reindex_project(
+                            {"id": project_id}
+                        )
+                        db.commit()
+                    else:
+                        # Get new session if db wasn't created
+                        db = next(get_db())
+                        db.execute(
+                            text("UPDATE codegraph_projects SET status = 'failed', updated_at = NOW() WHERE id = :id"),
+                            {"id": project_id}
+                        )
+                        db.commit()
+                except Exception as update_error:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)